src/PrestaShopBundle/Controller/Admin/Improve/International/LocalizationController.php

```diff
@@ -150,12 +213,32 @@ public function importPackAction(Request $request)
     }
 
     /**
-     * Returns localization settings form handler.
+     * Returns localization configuration form handler.
+     *
+     * @return FormHandlerInterface
+     */
+    private function getConfigurationFormHandler()
+    {
+        return $this->get('prestashop.admin.localization.configuration.form_handler');
+    }
+
+    /**
+     * Returns localization local units form handler.
+     *
+     * @return FormHandlerInterface
+     */
+    private function getLocalUnitsFormHandler()
```

Comment:
Maybe we can add the return type for every new method
    private function getLocalUnitsFormHandler(): FormHandlerInterface

Reply:
I think, it should be done in a new pull request. I just copy & paste original functions into classes :sweat_smile:  
I can change all files, depending on the decision the Core team will take ^^